test/interp/simd-shift.txt

```diff
@@ -40,6 +40,78 @@
     v128.const i32 0xff000055 0xe0000702 0xe0004003 0x00002004
     i32.const 67
     i64x2.shl)
+
```

Comment:
Could you make some of these tests show the effects of shifting outside the lane boundaries, as with the left shifts?

Reply:
Done.

Reply:
Thanks!